deploy/platform/kubernetes/templates/feature-event/permissions.yaml

```diff
@@ -32,5 +34,6 @@ roleRef:
   name: view
 subjects:
   - kind: ServiceAccount
-    namespace: ${NAMESPACE}
+    namespace: {{ .Release.Namespace }}
```

Comment:
Does the namespace need to be explicitly specified? This should be the default.

Reply:
> Does the namespace need to be explicitly specified? This should be the default.

I think you take it wrong, this `namespace` is in `subjects`, not in `metadata`, in `metadata` the default value is `{{ .Release.Namespace }}`, but in `subjects`, there is no default value for `namespace` and the `namespace` is required for `subjects` so we have to set it explicitly here